components/match2/commons/match_group_base.lua

```diff
@@ -6,31 +6,23 @@
 -- Please see https://github.com/Liquipedia/Lua-Modules to contribute
 --
 
-local p = {}
-
-local FeatureFlag = require('Module:FeatureFlag')
 local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
 local Match = require('Module:Match')
 local String = require('Module:StringUtils')
 local Variables = require('Module:Variables')
-local MatchGroupDisplay = require('Module:MatchGroup/Display')
-local getArgs = require('Module:Arguments').getArgs
 local json = require('Module:Json')
 
+local MatchGroupDisplay = Lua.import('Module:MatchGroup/Display', {requireDevIfEnabled = true})
+
 local category = ''
 local _loggedInWarning = ''
 
 local PARENT = Variables.varDefault('tournament_parent', '')
 
 local BRACKET_DATA_PARAMS = {'header', 'tolower', 'toupper', 'qualwin', 'quallose', 'skipround'}
 
--- Entry point used by Template:Matchlist
-function p.matchlist(frame)
```

Comment:
Could you explain why we no longer need these entry points?

Reply:
I changed the matchlist and bracket methods in Module:MatchGroup to call MatchGroup.luaMatchlist and MatchGroup.luaBracket. So the matchlist and bracket methods in Module:MatchGroup/Base aren't used anymore.

Reply:
you know that some wikis override `Module:MatchGroup`?

Reply:
Just valorant and one other I believe. The valorant one is accounted for here